site/public/js/forum.js

```diff
@@ -484,8 +497,21 @@ function alterShowMergeThreadStatus(newStatus, course) {
 }
 
 function modifyThreadList(currentThreadId, currentCategoriesId, course, loadFirstPage, success_callback){
-    var categories_value = $("#thread_category").val();
-    var thread_status_value = $("#thread_status_select").val();
+
+    var categories_value = [];
```

Comment:
This is a duplicate of line 380. Let's consolidate the code here.

Reply:
Moved to a function!